ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -262,8 +262,15 @@ define([
             }
         },
         deleteInvalidFilters: function(){
-            var currentValue = this.filterInput.currentView.model.getValue()[0];
-            if (currentValue === null){
+            const multiValueView = this.filterInput.currentView;
+            const valueCollectionView = multiValueView.values.currentView;
+            let allValid = true;
+            valueCollectionView.children.forEach(function (valueView){
```

Comment:
Can we do one of the alternatives to forEach that breaks out as soon as we know what we need to know?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some